src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,25 @@
+while (free W) {
+	move W
+}
+
+while (free N) {
+	move N
+}
+
+ while (free E) {
```

Comment:
Instead of asking the user to type the entire words, "blocked" and "free", would you have considered something simpler, like "b" for "blocked" or "f" for "free"?